docs/docs/deploying-to-21yunbox.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Deploying to 21YunBox
+---
+
+[21YunBox](https://www.21yunbox.com) provides blazing fast Chinese CDN, Continuous Deployment, one click HTTPS and [other services like managed databases and backend web services ](https://www.21yunbox.com/docs/v2/index.html), providing developers a hassle-free solution to launch their web projects in China.
+
+Static sites are **quick to deploy** on 21YunBox and include the following features:
+
+- Continuous, automatic builds & deploys from GitHub and Gitee.
+- Automatic SSL certificates through [Let's Encrypt](https://letsencrypt.org).
+- Instant cache invalidation with a blazing fast, Chinese CDN.
+- Unlimited [custom domains](https://www.21yunbox.com/docs/v2/custom-domains.html).
+- Automatic [Brotli compression](https://en.wikipedia.org/wiki/Brotli) for faster sites.
+- Native HTTP/2 support.
+- Automatic HTTP → HTTPS redirects.
+- Custom URL redirects and rewrites.
+
```

Comment:
Could you remove the periods at the end of these list items, please?

Reply:
Sure, already removed the periods on the latest commit.